README.md

```diff
@@ -1,3 +1,7 @@
 # Desktop Team Engineering Resources
 
 A collection of resources to standardise the Team's engineering processes and templates.
+
+## Troubleshooting
```

Comment:
I think we rather want to add this to the Troubleshooting section of the project-repo common template's README, no? (as one of the last items)

Reply:
Oops... Added to the wrong README, my bad.